test/unit/client-tests.js

```diff
@@ -674,135 +695,6 @@ describe('Client', function () {
       });
     });
   });
-  describe('#_setQueryOptions()', function () {
```

Comment:
Aren't these tests still applicable? (Other than the method names changing), or are things adequately covered by other tests?

Reply:
I think unit test of private methods should be used for functionality that is really hard to test, but this is not the case, unit tests on `test/unit/client-tests.js` and `test/unit/encoder-tests.js`and integration tests on `test/integration/short/load-balancing-tests.js` already cover the functionality.